src/html2md.js

```diff
@@ -75,11 +82,15 @@ function addMetadata(hast, mdast) {
   const head = select('head', hast);
   for (const child of head.children) {
     if (child.tagName === 'title') {
-      meta.set('title', toString(child));
+      meta.set(text('title'), text(toString(child)));
     } else if (child.tagName === 'meta') {
       const { name, content } = child.properties;
       if (name && !HELIX_META.has(name) && !name.startsWith('twitter:')) {
-        meta.set(name, content);
+        if (name === 'image') {
```

Comment:
there are more potential images: https://github.com/adobe/helix-html-pipeline/blob/e39efb089a583145894ebb79bfee8e3fe0f9fd0e/src/steps/extract-metadata.js#L266-L268

Reply:
But these are all derived from the `image` metadata no? Or can the author in doc-based authoring set a `og:image` as well as a `image`

In AEM we only set `image` when rendering the html for html2md.